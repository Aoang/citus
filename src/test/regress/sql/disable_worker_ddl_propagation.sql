SELECT success FROM run_command_on_workers('ALTER SYSTEM SET citus.enable_ddl_propagation TO OFF');
SELECT success FROM run_command_on_workers('select pg_reload_conf()');

