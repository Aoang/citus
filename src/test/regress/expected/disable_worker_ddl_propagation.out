SELECT success FROM run_command_on_workers('ALTER SYSTEM SET citus.enable_ddl_propagation TO OFF');
 success
---------------------------------------------------------------------
 t
 t
(2 rows)

SELECT success FROM run_command_on_workers('select pg_reload_conf()');
 success
---------------------------------------------------------------------
 t
 t
(2 rows)

